### DISCUSSION TITLE
I want to add a watermark in the background of a edit view.

### DISCUSSION THREAD

**Discussion by piyushdongre-dot-com** [filamentphp/filament#14290] [Help]
Hi I want to add something like "confidential" written in the background while someone editing the resources of a particular model. I am aware of creating a custom theme, but I just want it in the edit page and not anywhere else. Is that the way to go ?

Thanks.

**Comment by leandrocfe** [comment]
You can publish this view and customize what you need: https://filamentphp.com/docs/3.x/panels/resources/editing-records#custom-views
But take a look: https://filamentphp.com/docs/3.x/support/style-customization#publishing-blade-views

Another way that you can use to customize pages is using Render hooks
https://filamentphp.com/docs/3.x/support/render-hooks#overview
